docs/overview/tezos-different.md

```diff
@@ -20,12 +20,12 @@ Anyone who holds XTZ — the chain's native token — can propose changes to how
 
 ## Tezos uses proof of stake
 
-The proof-of-stake consensus model eliminates the need for high energy use, making it the "green" choice for blockchains. Instead of competing to achieve consensus as in proof-of-work models, Tezos nodes (called *bakers*) stake Tezos tokens to earn the right to create blocks and receive rewards. Users who want to participate without running a node themselves can delegate tokens to a baker for a share of the rewards. The bakers and delegators keep control of their tokens and can remove them at any time. Tezos's approach to consensus has been described as [Liquid Proof of Stake](https://medium.com/tezos/liquid-proof-of-stake-aec2f7ef1da7).
+The proof-of-stake consensus model eliminates the need for high energy use, making it the "green" choice for blockchains. Instead of competing to achieve consensus as in proof-of-work models, Tezos nodes (called *bakers*) stake Tezos tokens to earn the right to create blocks and receive rewards. Users who want to participate without running a node themselves can delegate tokens to a baker for a share of the rewards. The bakers and delegators keep control of their tokens: bakers can vary their staked amount in time, and delegators can spend their tokens or change their delegate at any time. Tezos's approach to consensus has been described as [Liquid Proof of Stake](https://medium.com/tezos/liquid-proof-of-stake-aec2f7ef1da7).
 
 The proof-of-stake model improves scalability and encourages incentive alignment. It also increases the cost of 51% attacks and avoids environmentally wasteful proof-of-work. Tezos launched in June 2018 as one of the first major proof-of-stake networks.
```

Comment:
I find that "encourages incentive alignment" is difficult to understand. Do you mean that incentives are designed to be aligned for the bakers (and delegators)? If so, it doesn't "encourage" but rather 'aims at" that, isn't it?

Reply:
Not sure what this is meant to mean. Perhaps it means that proof-of-work is a competitive model while the delegated proof of stake model allows users to pool resources and work together? Maybe it also means that the whole exercise is a collaborative effort between all bakers and delegators.

Reply:
OK, the what about being slightly more explicit about it, like:
```suggestion
The proof-of-stake model improves scalability and encourages cooperation through incentives alignment. It also increases the cost of 51% attacks and avoids environmentally wasteful proof-of-work. Tezos launched in June 2018 as one of the first major proof-of-stake networks.
```
(so let's avoid explicitly comparing to POW, because it seems that POW, beyond Bitcoin, can also be fixed for more incentives alignment, see <https://cs251.stanford.edu/lectures/lecture6.pdf>)

Reply:
I still think "incentives alignment" is cryptic. What about "encourages users to pool resources?"

Reply:
I do find this quite intuitive (once I googled around for it), byt it's true that I cannot find an accessible and neutral reference for this term. Then what about just "and encourages cooperation through incentives"? Just staying general (cooperation), and everyone knows what's an incentive.

(we could also add "well-designed incentives", but that's optional)

Reply:
"and encourages cooperation via incentives" sounds good to me.

Reply:
Done.